servers/graphql-kotlin-server/src/main/kotlin/com/expediagroup/graphql/server/extensions/InstrumentationExtensions.kt

```diff
@@ -0,0 +1,9 @@
+package com.expediagroup.graphql.server.extensions
+
+import com.expediagroup.graphql.dataloader.instrumentation.level.DataLoaderLevelDispatchedInstrumentation
+import com.expediagroup.graphql.dataloader.instrumentation.syncexhaustion.DataLoaderSyncExecutionExhaustedInstrumentation
+import graphql.execution.instrumentation.Instrumentation
+
+internal fun Instrumentation.isDataLoaderInstrumentation(): Boolean =
```

Comment:
guess it doesn't really matter but based on the naming conventions this file name should start with lowercase